docs/machine.md

```diff
@@ -0,0 +1,30 @@
+# Using with Docker Toolbox and Machine
+
+In development, we recommend using [Docker Toolbox](https://www.docker.com/products/docker-toolbox) to set up Docker. It includes a tool called Machine which will create a VM running Docker Engine and point your shell at it using environment variables.
```

Comment:
(Also – not all of them, but I talk to a lot of users so I have a pretty good sample size. ;)